src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -511,10 +520,10 @@ def getCEStatus(self):
         result["SubmittedJobs"] = 0
 
         magic = self.arcQueue + "_" + vo.lower()
-        for i in range(len(queueInfo)):
-            if queueInfo[i]["ID"].endswith(magic):
-                result["RunningJobs"] = int(queueInfo[i]["RunningJobs"])
-                result["WaitingJobs"] = int(queueInfo[i]["WaitingJobs"])
+        for _, qi in enumerate(queueInfo):
```

Comment:
Isn't this
```suggestion
        for qi in queueInfo:
```
?

Reply:
I think I just copied what was here: https://github.com/DIRACGrid/DIRAC/commit/6c317ceb47734ae8faa2725e0d0909123d914bf7#diff-c3ba3e0bcb7c5ba5048c51fabbd0ec40375efbda4078618d11f30d328eac5317L434

But indeed, you're right, I will correct that.